src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Predicates/PredicateQuery.cs

```diff
@@ -127,15 +127,15 @@ public string GetColumnName(string propertyPath)
                     {
                         // Switch the given alias in the path with the mapped alias.
                         // aliasPart.alias -> AliasPartIndex.Alias
-                        return Dialect.QuoteForTableName($"{tableAlias}", _configuration.Schema) + "." + Dialect.QuoteForColumnName(columnName);
+                        return $"{Dialect.QuoteForAliasName(tableAlias)}.{Dialect.QuoteForColumnName(columnName)}";
                     }
                 }
                 else
                 {
                     // no property provider exists; hope sql is case-insensitive (will break postgres; property providers must be supplied for postgres)
                     // Switch the given alias in the path with the mapped alias.
                     // aliasPart.Alias -> AliasPartIndex.alias
-                    return Dialect.QuoteForTableName($"{tableAlias}", _configuration.Schema) + "." + Dialect.QuoteForColumnName(values[0]);
+                    return $"{Dialect.QuoteForAliasName(tableAlias)}.{Dialect.QuoteForColumnName(values[0])}";
```

Comment:
```suggestion
                    return $"{Dialect.QuoteForAliasName(tableAlias)}.{Dialect.QuoteForColumnName(values.Last())}";
```
`values[0]` looks wrong to me as well, it has been changed in #14572 from `[1]`. I think it should be `Last()` here. Can you have a look?

Reply:
I noticed this issue that I thought should be changed as you mentioned, but I delayed it to another PR, I can include it here. agree?

Reply:
IMHO that is fine, any objections @hishamco?

Reply:
Done

Reply:
> IMHO that is fine, any objections @hishamco?

Nope